radix-engine/book/src/architecture/application/type_system/README.md

```diff
@@ -6,6 +6,6 @@ the Blueprint. This includes:
 * Function Input/Output 
 * Events
 
-The Type Checking System supports generics. 
+The Schema Checking System supports generics. 
```

Comment:
Slightly prefer to the previous word. Schema checking gives me the impression that we're checking the validity of schema (which is part of SBOR schema terminology)